### PR TITLE
:sparkles: [#1750] Add meta tags for social media sharing

### PR DESCRIPTION
**Pull request by stevenbal**

### CODE REVIEW

File: src/open_inwoner/pdc/views.py [hunk 1]
```diff
@@ -216,6 +216,7 @@ def get_context_data(self, **kwargs):
         if product.contacts.exists():
             anchors.append(("#contact", _("Contact")))
 
+        context["meta_description"] = product.summary
```
Comment (stevenbal) [inline src/open_inwoner/pdc/views.py:219]: @alextreme I'm not entirely sure if this is the right field to use as description in the meta tags, what do you think?

Comment (alextreme) [reply src/open_inwoner/pdc/views.py:219]: Using the product summary is fine

File: src/open_inwoner/templates/master.html [hunk 2]
```diff
@@ -4,6 +4,14 @@
         <meta charset="utf-8">
         <title>{% block title %}{% if page_title %}{{ page_title }} - {% endif %}{{ site_name }}{% endblock %}</title>
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <meta property="og:title" content="{% if page_title %}{{ page_title }} - {% endif %}{{ site_name }}">
```
Comment (alextreme) [inline src/open_inwoner/templates/master.html:8]: I would expect `<meta name="description" content="..."/>` here too

**Review by alextreme — CHANGES_REQUESTED** [review]
After adding name="description" this can be merged

**Review by Bartvaderkin — DISMISSED** [review]
I miss a few basic tags that matter and we have data available for, like og:image and twitter:image (every product has an image), og:url (product.get_absolute_url() eg without random query params) and og:type (hardcoded as 'article')

**Review by alextreme — APPROVED** [review]

**Comment by alextreme** [comment]
Mag gemerged worden zodra deze groen is

**Pull request merged**